pages/docs/reference/using-gradle.md

```diff
@@ -252,72 +357,128 @@ dependencies {
-    // or one of:
-    compile(kotlin("stdlib-jdk7"))
-    compile(kotlin("stdlib-jdk8"))
-}
+```kotlin
+compile(kotlin("stdlib-jdk7"))
+compile(kotlin("stdlib-jdk8"))
 ```
 
+</div>
 </div>
 
 In Kotlin 1.1.x, use `kotlin-stdlib-jre7` and `kotlin-stdlib-jre8` instead.
 
+If you target JavaScript, use the `stdlib-js` dependency.
+
+<div class="multi-language-sample" data-lang="groovy">
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-lang="groovy">
+
+```groovy
+compile "org.jetbrains.kotlin:kotlin-stdlib-js"
+```
+
+</div>
+</div>
+
+<div class="multi-language-sample" data-lang="kotlin">
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-lang="kotlin" data-highlight-only>
+
+```kotlin
+compile(kotlin("stdlib-js"))
+```
+
+</div>
+</div>
+
 If your project uses [Kotlin reflection](/api/latest/jvm/stdlib/kotlin.reflect.full/index.html) or testing facilities, you need to add the corresponding dependencies as well:
 
-<div class="sample" markdown="1" mode="groovy" theme="idea">
+<div class="multi-language-sample" data-lang="groovy">
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-lang="groovy">
 
 ```groovy
 compile "org.jetbrains.kotlin:kotlin-reflect"
 testCompile "org.jetbrains.kotlin:kotlin-test"
 testCompile "org.jetbrains.kotlin:kotlin-test-junit"
 ```
 
+</div>
 </div>
 
-Or, with Gradle Kotlin DSL:
-
-<div class="sample" markdown="1" mode="groovy" theme="idea">
+<div class="multi-language-sample" data-lang="kotlin">
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-lang="kotlin" data-highlight-only>
 
-```groovy
+```kotlin
 compile(kotlin("reflect"))
 testCompile(kotlin("test"))
 testCompile(kotlin("test-junit"))
 ```
 
+</div>
 </div>
 
 Starting with Kotlin 1.1.2, the dependencies with group `org.jetbrains.kotlin` are by default resolved with the version
-taken from the applied plugin. You can provide the version manually using the full dependency notation like
-`compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"`, or `kotlin("stdlib", kotlinVersion)` in Gradle Kotlin DSL.
+taken from the applied plugin. You can provide the version manually using the full dependency notation:
+ 
+<div class="multi-language-sample" data-lang="groovy">
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-lang="groovy">
+ 
+ ```groovy
+compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+ ```
+ 
+</div>
+</div>
+ 
+<div class="multi-language-sample" data-lang="kotlin">
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-lang="kotlin" data-highlight-only>
+ 
+ ```kotlin
+ compile(kotlin("stdlib", kotlinVersion))
+ ```
+ 
+</div>
+</div>
 
-## Annotation processing
+## Annotation Processing
 
-See the description of [Kotlin annotation processing tool](kapt.html) (`kapt`).
+Kotlin supports annonation processing via the _Kotlin annotation processing tool_(`kapt`). Usage of kapt with Gradle is described on the [kapt page](kapt.html).
```

Comment:
This link could contain an anchor to the section, like `[kapt page](kapt.html#gradle-build-cache-support-since-1220)` (not sure I correctly translated the title to an anchor, please check before adding it)

Reply:
+